### DISCUSSION TITLE
Chaine IPTV lien

### DISCUSSION THREAD

**Discussion by Letoumba01** [iptv-org/community#977] [Q&A]
Bonjour tous le monde , je recherche le lien des chaînes canal plus et Bein sport merci d'avance pour votre aide car je suis nous sur la plateforme

**Comment by PopeyeTheSai10r** [comment]
Vous pouvez utiliser https://iptv-org.github.io pour trouver un lien vers les chaînes

**Reply by Kobak1962** [reply]
Chaine en français de football⚽

**Comment by Darrel6322** [comment]
J'aimerais avoir plus de 30 mille chaînes dans mon iptv

**Comment by BellezaEmporium** [comment]
> Bonjour tous le monde , je recherche le lien des chaînes canal plus et Bein sport merci d'avance pour votre aide car je suis nous sur la plateforme

Ces chaînes-là sont payantes, en + d'avoir des chaînes bloquées (BeIN par exemple).

**Comment by alouest057** [comment]
bonjour je voudrais avoir un lien m3u  la chaine history.ca vf merci beaucoup

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
